<!DOCTYPE HTML>
<html>
	<head>
		<title>BBJ Lab at the University of Chicago</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper" class="divided">
				<!-- One -->
					<section class="banner style1 orient-left content-align-left image-position-right fullscreen onload-image-fade-in onload-content-fade-right">
						<div class="content">
							<div class="content-align-center"><img src="./images/logo.png" width="180"/></div>
							<p class="major">Machine Learning for Healthcare Research
								 at the University of Chicago</p>
							<ul class="actions stacked">
								<li><a href="#research" class="button big wide smooth-scroll-middle">Research</a></li>
								<li><a href="#people" class="button big wide smooth-scroll">People</a></li>
								<li><a href="#positions" class="button big wide smooth-scroll-middle">Open Positions</a></li>
								<li><a href="#email" class="button big wide smooth-scroll-middle">Get in touch</a></li>
							</ul>
						</div>
						<div class="image">
							<img src="images/banner-uc.jpg" alt="" />
						</div>
					</section>

					<!-- Two -->
					<section class="spotlight style1 orient-right content-align-left image-position-center onscroll-image-fade-in" id="research">
						<div class="content">
							<header>
								<h3>Primary Research Interests</h3>
							</header>
							<dl>
								<dt>1. Precision Phenotyping of Complex, Heterogenous Conditions </dt>
								<dd>
									<p>
										The presence of heterogeneity in complex conditions indicates the potential for both differing underlying biology / disease etiologies as well as environmental effects. We aim to use machine learning methods to better characterize phenotypes in order to exploit this heterogeneity to better understand biology and/or environmental factors of disease. 
									</p>
								</dd>
								<dt>2. Safe and Effective Deployment of Machine Learning in the Clinic </dt>
								<dd>
									<p>The rapid advancement of AI tools in clinical care demands robust strategies in implementation sciences, effective deployment, enhanced physician-computer interaction, and thorough impact assessment. As the creation of cutting-edge models becomes more prevalent, there's a tendency to overvalue their positive effects without adequately weighing the potential drawbacks, such as physician burnout and shifts in health equity. Our group is committed to devising tactics to refine models for specific healthcare environments, ensuring their seamless integration into clinical routines, and meticulously gauging their efficacy. Without rigorous assessments, including strategies like randomization, we cannot be sure these tools are leading to better healthcare. We are developing methods to address these and related problems.</p>
								</dd>
							</dl>
						</div>
						<div class="image">
							<img src="images/cluster-background.png" alt="" />
						</div>
					</section>

					<!-- Four -->
					<section class="spotlight style1 orient-left content-align-left image-position-center onscroll-image-fade-in" id="publications">
						<div class="content">
							<header>
								<h2>Select Publications</h2>
								<p><a href="https://scholar.google.com/citations?hl=en&user=2SvHNeQAAAAJ&view_op=list_works"> See a full list on Google Scholar </a></p>
							</header>
							<dl>
								<dt><a href="https://arxiv.org/abs/2504.08713">ProtoECGNet: Case-Based Interpretable Deep Learning for Multi-Label ECG Classification with Contrastive Learning</a></dt>
								<dd>
									<p>Sahil Sethi, David Chen, Thomas Statchen, Michael C. Burkhart, Nipun Bhandari, Bashar Ramadan, Brett K Beaulieu-Jones. <i>Preprint</i> (2025) 
									<p>ProtoECGNet is a prototype-based deep learning model for multi-label ECG classification that mirrors clinical interpretation by learning distinct prototypes for rhythm, morphology, and global abnormalities. It introduces a contrastive loss to structure the prototype space based on diagnostic co-occurrence, achieving near state-of-the-art performance on PTB-XL while providing faithful, case-based explanations grounded in real ECG segments from the training set. ProtoECGNet delivers transparent reasoning across all 71 labels, spanning a comprehensive range of cardiac abnormalities in 12-lead ECGs.
										 </p>
								</dd>
								<dt><a href="https://www.medrxiv.org/content/10.1101/2024.12.30.24319785v1">Advancing Healthcare AI Governance: A Comprehensive Maturity Model Based on Systematic Review</a></dt>
								<dd>
									<p>Rowan Hussein, Anna Zink, Bashar Ramadan, Frederick M Howard, Maia Hightower, Sachin Shah, Brett K Beaulieu-Jones. <i>Preprint</i> (2024) 
									<p>Our systematic analysis of healthcare AI governance frameworks revealed significant gaps in addressing diverse organizational needs, leading to the development of HAIRA - a novel, resource-aware maturity model spanning seven critical domains. This adaptive framework provides actionable governance pathways across five organizational levels, from small practices to major medical centers, enabling healthcare institutions to systematically assess and advance their AI governance capabilities based on available resources.
										 </p>
								</dd>
								<dt><a href="https://www.medrxiv.org/content/10.1101/2024.09.27.24314517v1">Synthetic Data Distillation Enables the Extraction of Clinical Information at Scale</a></dt>
								<dd>
									<p>Elizabeth Geena Woo*, Michael C Burkhart*, Emily Alsentzer, Brett Beaulieu-Jones. <i> Preprint</i> (2024) 
										<br> *co-first authors</p>
									<p>
										Our team demonstrated that synthetic data distillation can fine-tune smaller, open-source large-language models (LLMs) to achieve performance similar to larger models in extracting clinical information. This smaller model outperforms its base version and sometimes even the larger model. This approach will enable more scalable and cost-efficient clinical information extraction, improving tasks like patient phenotyping. </p>
								</dd>
								<dt><a href="https://www.nature.com/articles/s41531-024-00667-5"> Disease progression strikingly differs in research and real-world Parkinson’s populations </a></dt>
								<dd>
									<p> Brett K Beaulieu-Jones, Francesca Frau, Sylvie Bozzi, Karen J Chandross, M Judith Peterschmitt, Caroline Cohen, Catherine Coulovrat, Dinesh Kumar, Mark J Kruger, Scott L Lipnick, Lane Fitzsimmons, Isaac S Kohane, Clemens R Scherzer. <i>npj Parkinson's disease</i> (2024)</p>
									<p>
										Our team compared Parkinson's disease (PD) progression across research and real-world populations, utilizing real-world data (RWD) and large language models for detailed characterization. It finds that patients in real-world settings are diagnosed later and start treatment later than those in research populations, with faster motor and cognitive progression in real-world cohorts. The study highlights the differences between research and real-world populations, emphasizing the need to use diverse data sources and account for biases in clinical trial design and analyses. </p>
								</dd>
								<dt><a href="https://www.thelancet.com/journals/landig/article/PIIS2589-7500(23)00179-6/fulltext"> Predicting seizure recurrence after an initial seizure-like episode from routine clinical notes using large language models: a retrospective cohort study </a></dt>
								<dd>
									<p>Beaulieu-Jones, Brett K., Mauricio F Villamar, Phil Scordis, Ana Paula Bartmann, Waqar Ali, Benjamin D Wissel, Emily Alsentzer, Johann de Jong, Arijit Patra, Isaac Kohane. <i>Lancet Digital Health</i> (2023)</p>
									<p>
										Our team demonstrated machine learning models, particularly large language models pre-trained on domain-specific data, are highly effective in predicting seizure recurrence in children after an initial seizure-like event. These models outperformed traditional structured data approaches and indicate that clinical notes contain significant information useful for the prediction of seizure recurrence. </p>
								</dd>
								<dt><a href="https://www.medrxiv.org/content/10.1101/2024.04.18.24305861v1"> 	
								Phenotypic overlap between rare disease patients and variant carriers in a large population cohort informs biological mechanisms </a></dt>
								<dd>
									<p>Lane Fitzsimmons, Undiagnosed Diseases Network, Brett Beaulieu-Jones*, Shilpa Nadimpalli Kobren* <i>Preprint (in press)</i> (2024) <br> 
										*co-corresponding</p>
									<p>
										The biological mechanisms causing extreme symptoms in rare disease patients are complex and often elusive. This study analyzes genotype and phenotype data from the UK Biobank to understand the pathways leading to seizures in undiagnosed patients from the Undiagnosed Diseases Network. By examining milder, related symptoms in UK Biobank participants with similar genetic variants, the study aims to shed light on the molecular mechanisms behind these rare conditions 
									</p>
								</dd>
								<dt><a href="https://www.medrxiv.org/content/10.1101/2024.09.15.24313708v1"> Characterizing the connection between Parkinson's disease progression and healthcare utilization </a></dt>
								<dd>
									<p>Lane Fitzsimmons, Francesca Frau, Sylvie Bozzi, Karen Chandross, Brett Beaulieu-Jones <i> Preprint </i> (2024)</p>
									<p>
										This study analyzed Parkinson's disease (PD) progression by examining clinical events across different Hoehn & Yahr (H&Y) stages extracted using natural language processing. It provides a view of healthcare utilization at different H&Y stages, models expected H&Y progression and demonstrates the potential value for a therapeutic which would slow progression.  </p>
								</dd>
								<dt><a href="https://www.nature.com/articles/s41746-021-00426-3">Machine learning for patient risk stratification: standing on, or looking over, the shoulders of clinicians?</a></dt>
								<dd>
									<p>Beaulieu-Jones, Brett K., William Yuan, Gabriel A. Brat, Andrew L. Beam, Griffin Weber, Marshall Ruffin, and Isaac S. Kohane.  <i>NPJ digital medicine</i> (2021)</p>
									<p>
										We trained deep learning models on clinician-initiated administrative data for 42.9 million admissions and found performance close to full EMR-based benchmarks for inpatient outcomes. These models rely heavily on clinical behavior, and should not be used for individualized clinical decision making. For meaningful clinical guidance, models should outperform these benchmarks using data sources that capture patient state rather than clinician actions (i.e., looking over their shoulder). </p>
								</dd>
								<dt><a href="https://ascpt.onlinelibrary.wiley.com/doi/full/10.1002/cpt.1658">Examining the use of real‐world evidence in the regulatory process</a></dt>
								<dd>
									<p>Beaulieu‐Jones, Brett K., Samuel G. Finlayson, William Yuan, Russ B. Altman, Isaac S. Kohane, Vinay Prasad, and Kun‐Hsing Yu. <i> Clinical Pharmacology & Therapeutics</i> (2020)</p>
									<p>
										The 21st Century Cures Act requires the US FDA to create guidelines for using real-world evidence (RWE) in the regulatory process. While RWE has led to crucial medical findings, it faces challenges in proving treatment efficacy compared to randomized controlled trials. In this review article, we summarized the advantages and limitations of RWE, identified the key opportunities for RWE, and pointed the way forward to maximize the potential of RWE for regulatory purposes.</p>
								</dd>
								<dt><a href="https://www.ahajournals.org/doi/full/10.1161/CIRCOUTCOMES.118.005122">Privacy-preserving generative deep neural networks support clinical data sharing</a></dt>
								<dd>
									<p>Beaulieu-Jones, Brett K., Zhiwei Steven Wu, Chris Williams, Ran Lee, Sanjeev P. Bhavnani, James Brian Byrd, and Casey S. Greene. <i>Circulation: Cardiovascular Quality and Outcomes</i> (2019)</p>
									<p>
										Our team has developed a method using deep neural networks to generate synthetic data that closely resembles real participants from the SPRINT trial, ensuring privacy while maintaining the utility of the data for research. This technique allows for the sharing of clinical data with researchers for secondary analysis without risking patient privacy.</p>
								</dd>
								<dt><a href="https://www.nature.com/articles/nbt.3780">Reproducibility of computational workflows is automated using continuous analysis</a></dt>
								<dd>
									<p>Beaulieu-Jones, Brett K., and Casey S. Greene. <i>Nature biotechnology</i> (2017)</p>
									<p>
										Continuous analysis is a workflow that integrates Docker container technology with continuous integration to automatically rerun computational analyses upon any changes in source code or data. This approach facilitates effortless reproducibility of research results for peers and provides an audit trail for data analyses, enhancing transparency and reliability in scientific studies.
									</p>
								</dd>

								<dt><a href="https://www.sciencedirect.com/science/article/pii/S153204641630140X">Semi-supervised learning of the electronic health record for phenotype stratification</a></dt>
								<dd>
									<p>Beaulieu-Jones, Brett K., and Casey S. Greene. <i>Journal of biomedical informatics</i> (2016)</p>
									<p>
										We developed a semi-supervised learning technique to improve the extraction of phenotypes from electronic health records, aiding in the identification of disease subtypes and genetic associations. This method has shown promise in enhancing classification accuracy and predicting patient outcomes, even with limited high-quality data.</p>
								</dd>
							</dl>
						</div>
					</section>

					<!-- Six -->
					<section class="wrapper style1 align-center" id="people">
						<div class="inner">
							<h2>People</h2>
							<p>We're always looking to add talented & curious students, post-docs, programmers and data scientists.</p>
							<div class="items style1 medium onscroll-fade-in">
								<section>
									<span class="icon style2 major"><img src="./headshots/bbj-circle-140.png"></img></span>
									<h3>Brett Beaulieu-Jones, PhD</h3>
									<p>Assistant Professor</p>
									<p>Sections of Biomedical Data Science & Genetic Medicine</p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/mcl-circle-140.png"></img></span>
									<h3>Ming-Chieh (Eddie) Liu, MS</h3>
									<p>Research Associate</p>
									<p>Research associate examining the impact and potential of diverging predictions between different large-scale clinical AI models for the same patients.</p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/bashar-circle-140.png"></img></span>
									<h3>Bashar Ramadan, MBBS</h3>
									<p>Attending Physician</p>
									<p>Bashar provides the clinical expertise for numerous projects in the group - especially in evaluating divergence between predictive models.</p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/david-circle-140.png"></img></span>
									<h3>David Chen, MS</h3>
									<p>Data Scientist</p>
									<p> David is working on "Argos" a project developing tools and methods to make the evaluation and monitoring of Clinical AI easier. </p>
								</section>
								<section>
									<a href="https://zinka88.github.io/">
										<span class="icon style2 major"><img src="./headshots/zink-circle-140.png"></img></span>
										<h3>Anna Zink, PhD </h3>
									</a>
									<p>Principal Researcher (Center for Applied AI - Booth)</p>
									<p>Anna is working closely with the lab on a multi-site project to evaluate the performance and fairness of machine learning models in the clinic and to develop scalable frameworks for this going forward.</p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/geena-circle-140.png"></img></span>
									<h3>Geena Woo, BA</h3>
									<p>PhD Candidate (co-advised with Gilad Lab)</p>
									<p>Geena is using machine learning and causal inferences approaches to better understand the impact of maternal exposures on their children.</p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/michael-circle-140.png"></img></span>
									<h3>Michael Burkhart, PhD</h3>
									<p>Senior Data Scientist</p>
									<p>Michael is working on multiple projects to improve clinical information extraction and to better understand foundational clinical AI models.</p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/rowan-circle-140.png"></img></span>
									<h3>Rowan Hussein, BA</h3>
									<p>Currently: Med Student</p>
									<p>Rowan is working to harmonize governance frameworks in order to provide a practical and actionable roadmap for health systems. </p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/tom-circle-140.png"></img></span>
									<h3>Tom Statchen, BS</h3>
									<p>Currently: Med Student  </p>
									<p>Tom is working to evaluate the performance and failure cases of deployed generative AI systems in the health system. </p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/sahil-circle-140.png"></img></span>
									<h3>Sahil Sethi, BS</h3>
									<p>Currently: Med Student </p>
									<p>Sahil is working on deep learning methods for medical time-series applications (eg, ECGs), with a focus on enhancing model interpretability and improving decision aids. </p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/krish-circle-140.png"></img></span>
									<h3>Krish Shah, BS</h3>
									<p>Currently: MS Student - Biomedical Data Science</p>
									<p>Krish is developing machine learning and statistical methods to better understand disease progression.</p>
								</section>
							</div>
						</div>
						<div class="inner">
							<h2>Alumni, Close Collaborators & Thesis Examinees</h2>
							<div class="items style1 medium onscroll-fade-in">
								<section>
									<span class="icon style2 major"><img src="./headshots/sylvia-circle-140.png"></img></span>
									<h3>Sylvia Edoigiawerie, PhD </h3>
									<p>Currently: Completing the MD portion of her MD/PhD at UChicago  </p>
									<p>Sylvia completed her PhD with Dr. Naoum Issa and collaborates closely with the lab for her projects developing methods to improve seizure detection of neonates. </p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/lane-circle-140.png"></img></span>
									<h3>Lane Fitzsimmons, BS</h3>
									<p>Lane was a research associate and continues to be a collaborator. She is currently a Med student at the Renaissance School of Medicine at Stony Brook University. </p>
								</section>
								<section>
										<span class="icon style2 major"><img src="./headshots/temi-circle-140.png"></img></span>
										<h3>Temidayo Adeluwa, MS</h3>
									<p> Temi is a PhD Candidate in the Genetics, Genomics and Systems Biology Program at UChicago with Dr. Haky Im and I'm the chair of his thesis committee.</p>
								</section>
								<section>
									<a href="https://www.linkedin.com/in/jessica-karina-de-freitas/">
										<span class="icon style2 major"><img src="./headshots/jess-circle-140.png"></img></span>
										<h3>Jessica De Freitas, PhD</h3>
									</a>
									<p> I served as Jess's external examiner for her thesis on "Leveraging Electronic Health Records and Electrocardiograms for Disease Phenotyping" in 2023. She is currently a Machine Learning Scientist at Tempus Labs, Inc. </p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/yidi-circle-140.png"></img></span>
									<h3>Yidi Huang, MS </h3>
									<p>Yidi was a research associate and is currently a PhD Student in the program I graduated from at the University of Pennsylvania. (Genomics and Computational Biology)</p>
								</section>
								<section>
									<span class="icon style2 major"><img src="./headshots/saqib-circle-140.png"></img></span>
									<h3>Mohammed Saqib, BS </h3>
									<p> Mohammed was a research associate and is currently a PhD Student at the University of Pennsylvania, where he's working on neuroimaging.</p>
								</section>
								
							</div>
						</div>
					</section>

					<section class="wrapper style1 align-center" id="positions">
						<div class="inner medium">
							<h2>Open Positions </h2>
							<p> We are actively recruiting multiple students, postdocs and/or data scientists. <a href="#email">Get in touch</a> if you're driven to work on these problems, or propose your own ideas tied to our <a href="#research">research interests.</a> When reaching out it is incredibly helpful if you specify what led you to reaching out and to tell me about how our interests overlap. I'm open to creative extensions of the research interests listed above (e.g., if you are interested in a different area but would be using similar methods etc.). </p>
							<img src="images/email.jpg" alt=""/>
						</div>
					</section>

					<section class="wrapper style1 align-center">
						<div class="inner">
							<h2>Active / Recent Funding and Support </h2>
							<p> We are extremely grateful for the organizations who support and have supported our work!</p>
							<div class="items style1 medium onscroll-fade-in">
								<section>
									<img src="./images/funders/ninds-logo.png" width="225"/>
								</section>
								<section>
									<img src="./images/funders/gbm.png" width="225"/>
								</section>
								<section>
									<img src="./images/funders/sanofi.png" width="225"/>
								</section>
								<section>
									<img src="./images/funders/ucb.webp" width="225"/>
								</section>
								<section>
									<img src="./images/funders/merck.webp" width="225"/>
								</section>
						</div>

				<!-- Seven -->
					<section class="wrapper style1 align-center" id="email">
						<div class="inner medium">
							<h2>Get in touch</h2>
							<img src="images/email.jpg" alt="" />
						</div>
					</section>

				<!-- Footer -->
					<footer class="wrapper style1 align-center">
						<div class="inner">
							<ul class="icons">
								<li><a href="https://twitter.com/beaulieujones" class="icon brands style2 fa-twitter"><span class="label">Twitter</span></a></li>
							</ul>
							<p>&copy; <a href="./bio.html">Brett Beaulieu-Jones.</a> Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
						</div>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
